docs/guides/parquet-mode.md

```diff
@@ -0,0 +1,244 @@
+2. **Conversion Status**: Blocks are only converted once, tracked via marker files
+3. **Tenant Settings**: Conversion must be enabled for the specific tenant
+
+The conversion process:
+- Downloads TSDB blocks from object storage
+- Converts time series data to Parquet format
+- Uploads Parquet files (chunks and labels) to object storage
+- Creates conversion marker files to track completion
+
+## Querying Behavior
+
+When parquet queryable is enabled:
+
+1. **Primary Query Path**: Attempts to query Parquet files first
+2. **Fallback Logic**: Falls back to TSDB blocks if the Parquet files are not available - block hasn't been converted yet
+3. **Hybrid Queries**: Can query both Parquet and TSDB blocks in the same query
+
+## Monitoring
+
+### Parquet Converter Metrics
+
+Monitor parquet converter operations:
+
+```promql
+# Blocks converted
+cortex_parquet_converter_blocks_converted_total
+
+# Conversion failures
+cortex_parquet_converter_block_convert_failures_total
+
+# Delay in minutes of Parquet block to be converted from the TSDB block being uploaded to object store
+cortex_parquet_converter_convert_block_delay_minutes
+```
+
+### Parquet Queryable Metrics
+
+Monitor parquet query performance:
+
+```promql
+# Blocks queried by type
+cortex_parquet_queryable_blocks_queried_total
+
+# Query operations
+cortex_parquet_queryable_operations_total
+
+# Cache metrics
+cortex_parquet_queryable_cache_hits_total
+cortex_parquet_queryable_cache_misses_total
+```
+
+## Best Practices
+
+### Deployment Recommendations
+
+1. **Dedicated Converters**: Run parquet converters on dedicated instances for better resource isolation
+2. **Ring Configuration**: Use a distributed ring for high availability and load distribution
+3. **Storage Considerations**: Ensure sufficient disk space in `data_dir` for block processing
+4. **Network Bandwidth**: Consider network bandwidth for downloading/uploading blocks
+
+### Performance Tuning
+
+1. **Row Group Size**: Adjust `max_rows_per_row_group` based on your query patterns
+2. **Cache Size**: Tune `parquet_queryable_shard_cache_size` based on available memory
+3. **Concurrency**: Adjust `meta_sync_concurrency` based on object storage performance
+4. **Conversion Interval**: Balance between conversion latency and system load
+
+### Query Optimization
+
+1. **Time Range Queries**: Parquet performs best with time-range based queries
+2. **Label Filtering**: Use label filters to reduce data scanning
+3. **Aggregation**: Leverage Parquet's columnar format for aggregation queries
+
+## Limitations
+
+1. **Experimental Feature**: Parquet mode is experimental and may have stability issues
+2. **Storage Overhead**: Parquet files are stored in addition to TSDB blocks
+3. **Conversion Latency**: There's a delay between block creation and parquet availability
+4. **Query Compatibility**: Some advanced PromQL features may not be fully supported
```

Comment:
Two limitations I can think of:
- It doesn't support non shuffle sharding mode
- Bucket Index is required to work with Parquet query.

Reply:
PTAL?